src/components/footer.tsx

```
@@ -1,4 +1,5 @@
 import { Activity, Code } from "lucide-react"
+import Image from "next/image"
```

ESLint

[error] Strings must use singlequote.

(@stylistic/quotes)